gramps/gui/glade/editplace.glade

```diff
@@ -235,7 +235,7 @@ You can set these values via the Geography View by searching the place, or via a
                       <object class="ValidatableMaskedEntry" id="latlon_entry">
                         <property name="visible">True</property>
                         <property name="can_focus">True</property>
-                        <property name="tooltip_text" translatable="yes">Field used to paste info from a web page like google, openstreetmap, ... </property>
+                        <property name="tooltip_text" translatable="yes">Field used to paste info from a web page like OpenStreetMap, Google, ...</property>
```

Comment:
This is addressed in https://github.com/gramps-project/gramps/pull/1213

Reply:
:(
I fork project today and this fix not be present in code.